addon/snapshot.js

```diff
@@ -1,6 +1,18 @@
-import Ember from 'ember';
 import { getNativeXhr } from './native-xhr';
 import { maybeDisableMockjax, maybeResetMockjax } from './mockjax-wrapper';
+import ajax from 'ember-fetch/ajax';
+import run from 'ember-runloop';
```

Comment:
I'm kind of confused as to why this works, and can't find any references to doing it this way. Is there a preference for this over destructuring `run` off of the `Ember` object?

Reply:
This is working because [ember-cli-shims](https://github.com/ember-cli/ember-cli-shims) is included as a dependency. Destructuring from the `Ember` object is now generally not recommended.

Ember is slowly moving towards using separate packages, so the preference is to import from these packages so that when future improvements are added to the build process (tree-shaking with [rollup](https://github.com/rollup/rollup)) it will result in smaller build files.

`import run from 'ember-runloop';` then becomes `import { run } from '@ember/runloop';` and a list of these new module imports can be found here: https://github.com/ember-cli/ember-rfc176-data

The documentation and guides are also in the process of being updated as of Ember@2.16 and you can see the beginning of this: https://emberjs.com/api/ember/2.16/classes/@ember%2Frunloop

Reply:
You're welcome, glad that was helpful. No worries, it's a lot of work to stay up to date. 😄